<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmIdle</class>
 <widget class="QWidget" name="ConfirmIdle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置确认空闲窗口</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>信号设备</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="1" column="0" rowspan="2" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>选择类型</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>确认空闲</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
